ui/app/components/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -131,20 +131,26 @@ export default class ImportWithSeedPhrase extends PureComponent {
     return !passwordError && !confirmPasswordError && !seedPhraseError
   }
 
+  toggleTermsCheck = () => {
+    this.setState({
```

Comment:
We should prefer [the `updater` function argument](https://reactjs.org/docs/react-component.html#setstate) here. That is,

```js
this.setState((prevState) => ({
    termsChecked: !prevState.termsChecked,
})
```

Reply:
Done in ac872717b